src/sql/base/browser/ui/taskbar/collapsibleActionbar.ts

```diff
@@ -0,0 +1,294 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IAction } from 'vs/base/common/actions';
+import { StandardKeyboardEvent } from 'vs/base/browser/keyboardEvent';
+import { KeyCode, KeyMod } from 'vs/base/common/keyCodes';
+import {
+	IActionBarOptions, ActionsOrientation, IActionViewItem,
+	IActionOptions
+} from 'vs/base/browser/ui/actionbar/actionbar';
+import * as DOM from 'vs/base/browser/dom';
+import * as types from 'vs/base/common/types';
+import * as nls from 'vs/nls';
+import { debounce } from 'vs/base/common/decorators';
+import { ActionBar } from 'sql/base/browser/ui/taskbar/actionbar';
+
+const defaultOptions: IActionBarOptions = {
+	orientation: ActionsOrientation.HORIZONTAL,
+	context: null
+};
+
+export class CollapsibleActionBar extends ActionBar {
```

Comment:
Maybe just call it OverflowActionBar or something? Collapsible to me seems to imply it collapses - i.e. gets smaller - which isn't exactly what's happening.

Reply:
renamed to OverflowActionBar